MapleServer2/Managers/FieldManager.cs

```diff
@@ -1363,7 +1473,7 @@ private void HealingSpot()
                     continue;
                 }
 
-                player.AdditionalEffects.AddEffect(new(70000018, 1)); // applies a healing effect to the player
+                player.TaskScheduler.QueueBufferedTask(() => player.AdditionalEffects.AddEffect(new(70000018, 1))); // applies a healing effect to the player
```

Comment:
Not totally related to your PR, but I think I've seen where the bull ID comes from and a better way to handle field spots that isn't specifically a healing spot. It's in the xblocks IIRC

Reply:
Ohh alright. Yeah this could definitely be changed to use xblock parsing instead of hard coding.